internal/handlers/common/aggregations/count.go

```diff
@@ -77,6 +77,23 @@ func newCount(stage *types.Document) (Stage, error) {
 	}, nil
 }
 
+// countAccumulator represents $count accumulator for $group.
+type countAccumulator struct{}
```

Comment:
`$group`'s `$count` accumulator should not be same file as `$count` stage. They are completely unrelated.